packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -55,6 +55,11 @@ jest.mock('uuid', () => {
   };
 });
 
+jest.mock('ulidx', () => ({
+  ulid: () => 'ULID01234567890ABCDEFGHIJKLMN',
+  monotonicFactory: () => () => 'ULID01234567890ABCDEFGHIJKLMN',
+}));
```

Comment:
Instead of mocking here and assigning the same ID to all keyrings, perhaps we can mock it at test case level so we can test additional functionality